std_msgs/String task
---
zeabus_vision/vision_dice data